test/it/WatcherPackageIT.ts

```diff
@@ -1,14 +1,105 @@
 import { params, skip, slow, suite, timeout } from "../../index";
 import { AbstractPackageITBase, PackageTestParams } from "./AbstractPackageITBase";
 
-import { spawn } from "child_process";
+import { ChildProcess, spawn } from "child_process";
 
-@suite(timeout(90000), slow(10000))
+import * as assert from "assert";
+import { existsSync, unlinkSync, writeFileSync } from "fs";
+import { join, resolve } from "path";
+import * as readline from "readline";
+
+const editFilePath = join(__dirname, "fixtures", "packages", "watcher", "test", "new.ts");
+const cwd = resolve(__dirname, "fixtures", "packages", "watcher");
+const shell = true;
+
+@suite(timeout(90000), slow(90000))
 class WatcherPackage extends AbstractPackageITBase {
+
+  watch: ChildProcess;
+  readline: readline.Interface;
+
   @params({ fixture: "watcher", installTypesMocha: false }, "can run watcher")
   @params({ fixture: "watcher", installTypesMocha: true }, "can run watcher with @types/mocha")
-  runTest(params: PackageTestParams) {
+  async runTest(params: PackageTestParams) {
     super.runTest(params);
```

Comment:
I want to be sure that if it fails it will be because of the watcher issues, not because of issues with mocha-typescript. Further the watcher will need the dependencies installed I think that's handled in the super call.